contributing-device-support.md

```diff
@@ -0,0 +1,66 @@
+# Contributing new board support to resin.io
+
+Pre-requisites: yocto BSP layer for your particular board, supplying the BSP (board support package). Should be compatible to yocto daisy/fido/jethro release at the time of writing this howto.
```

Comment:
I'd remove `at the time of writing this howto.` the Pre-requisites shouldn't change that frequently and if they do we should update the doc.